topics/t3-stack/index.md

```diff
@@ -0,0 +1,18 @@
+---
+aliases: t3, t3-stack
```

Comment:
You can't use the topic as an alias for itself 😇 . This should get CI green:

```suggestion
aliases: t3
```

Reply:
Thanks!